src/main/java/io/gingersnapproject/cdc/configuration/DefaultRuleParser.java

```diff
@@ -0,0 +1,148 @@
+package io.gingersnapproject.cdc.configuration;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.infinispan.commons.dataconversion.internal.Json;
+
+public final class DefaultRuleParser {
```

Comment:
Ultimately we need something that's more flexible than this for @rigazilla work consuming EagerCacheRules via the ConfigMap.

At deployment time, the db-syncer will be configured so that it's application.properties contains all connection information for the DB and cache service. It will just be individual rules that CRUD operations are performed on at runtime.

Reply:
The approach I took here was to avoid adding more dependencies or using Jackson, but sure enough, this is super specific for the current object and will need updates in case of add/remove properties. If it's OK we can put this on hold until that one, so we can create something more adequate, especially the parsing bits. WDYT? As I don't think this one is crucial for our current work.

I'll apply the other suggestions.

Reply:
> If it's OK we can put this on hold until that one, so we can create something more adequate, especially the parsing bits. WDYT?

+1, there's no point spending time on this here if it's going to be invalidated by @rigazilla coming work.